ocs_ci/deployment/deployment.py

```diff
@@ -4234,7 +4252,12 @@ def deploy(self):
         odf_running_version = version.get_semantic_ocs_version_from_config()
         if odf_running_version >= version.VERSION_4_19:
             # create service exporter
-            create_service_exporter()
+            for cluster in get_non_acm_cluster_config():
```

Comment:
Please skip client clusters

Reply:
let me add confiton to skip for service_exporter